src/classes/CSVBatchProcess.cls

```diff
@@ -0,0 +1,68 @@
+/**
+* File Name :   CSVBatchProcess.cls
+* Description : Class processing iported CSV file in batches
+*               
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 20/05/2014 Alex PAnka     Initial version
+*
+*
+*/
+global with sharing class CSVBatchProcess implements Database.Batchable<sObject>{
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.